src/main/java/com/commafeed/backend/feed/FeedParser.java

```diff
@@ -160,8 +160,8 @@ private Date validateDate(Date date, boolean nullToNow) {
 		}
 		return date;
 	}
-
-	private String getContent(SyndEntry item) {
+	// This function have been changed to protected for testing purposes

```

Comment:
Same as above

Reply:
I will get it as a question for the teacher next class...